spec/services/notifications/journal_wp_notification_service_spec.rb

```diff
@@ -221,13 +221,23 @@ def call
         ]
       end
 
-      # Event creation will be prevented by the service
+      it_behaves_like 'creates no notification'
+    end
+
+    context 'assignee has all in app notifications enabled but only involved for mail' do
```

Comment:
Sorry, but that is just getting too long for me. While I understand the desire for clarity, I don't think that it can be achieved completely in the `context` part. In #9430 even more variables are included in the spec and if I name them all, this would become quite a long text. There will always be the need to read the code, I believe.